packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -0,0 +1,386 @@
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/utils/SafeCast.sol";
+
+import "./interfaces/IFederatedAttestations.sol";
+import "../common/interfaces/IAccounts.sol";
+import "../common/interfaces/ICeloVersionedContract.sol";
+
+import "../common/Initializable.sol";
+import "../common/UsingRegistry.sol";
+import "../common/Signatures.sol";
+import "../common/UsingPrecompiles.sol";
+import "../common/libraries/ReentrancyGuard.sol";
+
+/**
+ * @title Contract mapping identifiers to accounts
+ */
+contract FederatedAttestations is
+  IFederatedAttestations,
+  ICeloVersionedContract,
+  Ownable,
+  Initializable,
+  UsingRegistry,
+  ReentrancyGuard,
+  UsingPrecompiles
+{
+  using SafeMath for uint256;
+  using SafeCast for uint256;
+
+  struct OwnershipAttestation {
+    address account;
+    uint256 issuedOn;
+    address signer;
+  }
+
+  // TODO ASv2 revisit linting issues & all solhint-disable-next-line max-line-length
```

Comment:
Note: If you want to ensure something is addressed before the PR is merged using the string `DO NOT MERGE` can be useful because there is a CI check to prevent any PR with that string included from being merged.

Reply:
Ooh that's a great tip! Will update the TODOs 😄

Reply:
(update didn't change to that because this sadly breaks the lint checks which then blocks a bunch of other tests in CI 😬)